src/inabox/inabox-resources.js

```diff
@@ -189,11 +186,6 @@ export class InaboxResources {
     return this.firstPassDone_.promise;
   }
 
-  /** @override */
-  isIntersectionExperimentOn() {
```

Comment:
Does this mean inabox-resources will act differently now?

Reply:
I believe the only reason this method was added was to satisfy `resources-interface`, but it was essentially a no-op.